apis/actions.md

```diff
@@ -0,0 +1,125 @@
+---
+description: Creating tool items, GLib.Actions, and keyboard shortcuts
+---
+
+# Actions
+
+GTK and GLib have a powerful API called [`GLib.Action`](https://valadoc.org/gio-2.0/GLib.Action.html) which can be used to define the primary actions of your app, assign them keyboard shortcuts, and tie them to [`Actionable`](https://valadoc.org/gtk+-3.0/Gtk.Actionable.html) widgets in your app like Buttons and Menu Items. In this section, we're going to create a Quit action for your app with an assigned keyboard shortcut and a Button that shows that shortcut in a tooltip. Begin by creating a `Gtk.Application` with a `Gtk.ApplicationWindow` as you've done in previous examples.
```

Comment:
New paragraph at "Begin by creating a ....", since we have moved from the introduction to actually implementing the example.